static/src/components/builder/components/ComponentRender/ComponentRender.vue

```diff
@@ -156,6 +156,11 @@ export default {
             }
           }
         };
+      } else if (this.selectedComponent === 'task-timer') {
+        return {
+          name: 'task-timer',
+          attrs: { lock: `${this.form.showLock}` }
```

Comment:
The name "lock" and "showLock" here as well in the "task-timer" component seem a little misleading. Shall we use some names like "clock" or "lockTime" or "time"?